reference/api/webhooks.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Webhooks
+description: Use the /webhooks to trigger automatic workflows when Meilisearch finishes processing tasks.
+---
+
+import { RouteHighlighter } from '/snippets/route_highlighter.mdx'
+
+import CodeSamplesWebhooksGet1 from '/snippets/samples/code_samples_webhooks_get_1.mdx';
+import CodeSamplesWebhooksPost1 from '/snippets/samples/code_samples_webhooks_post_1.mdx';
+import CodeSamplesWebhooksPatch1 from '/snippets/samples/code_samples_webhooks_patch_1.mdx';
+import CodeSamplesWebhooksDelete1 from '/snippets/samples/code_samples_webhooks_delete_1.mdx';
+
+Use the `/webhooks` to trigger automatic workflows when Meilisearch finishes processing tasks.
+
+## The webhook object
+
+```json
+{
+  "uuid": "V4_UUID_GENERATED_BY_MEILISEARCH",
+  "url": "WEBHOOK_NOTIFICATION_TARGET_URL",
+  "headers": {
+    "HEADER": "VALUE",
+  },
+  "isEditable": false
+}
+```
+
+- `uuid`: a v4 uuid Meilisearch automatically generates when you create a new webhook
+- `url`: a string indication the URL Meilisearch should notify whenever it completes a task, required
+- `headers`: an object with HTTP headers and their values, optional, often used for authentication
+- `isEditable`: read-only Boolean field indicating whether you can edit the webhook. Meilisearch automatically sets this to false for its internal webhooks
```

Comment:
```suggestion
- `isEditable`: read-only Boolean field indicating whether you can edit the webhook. This can only be set automatically by Meilisearch for its internal webhooks
```